addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.js

```diff
@@ -16,12 +44,20 @@ export class ThreadViewTopbar extends Component {
         useRefToModel({ fieldName: 'threadNameInputRef', modelName: 'mail.thread_view_topbar', propNameAsRecordLocalId: 'localId', refName: 'threadNameInput' });
         useRefToModel({ fieldName: 'threadDescriptionInputRef', modelName: 'mail.thread_view_topbar', propNameAsRecordLocalId: 'localId', refName: 'threadDescriptionInput' });
         useUpdateToModel({ methodName: 'onComponentUpdate', modelName: 'mail.thread_view_topbar', propNameAsRecordLocalId: 'localId' });
+        this._inviteButtonRef = useRef('inviteRef');
     }
 
     //--------------------------------------------------------------------------
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @returns {HTMLElement}
+     */
+    getInvitePopoverTarget() {
```

Comment:
overkill indirection, but with my suggestion above you'll likely need to change it to a useRefToModel

Reply:
The hook `usePosition` is low-level and requires `target` to be an HTML element or a function returning an HTML element. I suppose it's used to determine position of the popover, and the `web.Popover` version didn't need it by means of button being wrapped as slot (and target being deduced that way).

The proper way would be to make model aware of UI positioning, but this looks too complex to do in this PR, especially when the workaround is small and safe enough for the time being.

Reply:
Could slightly improve by turning `target` prop as a popover model field, and set it to ref of invite button in model.